Source/Xamarin/Prism.Unity.Forms/PrismApplication.cs

```diff
@@ -17,6 +17,8 @@ namespace Prism.Unity
 {
     public abstract class PrismApplication : PrismApplicationBase
     {
+        const string _navigationServiceName = "UnityPageNavigationService";
```

Comment:
Minor, but are we sure we don't want this public or at least protected so it can be more easily referenced?